react/features/base/conference/reducer.ts

```diff
@@ -60,7 +60,7 @@ export interface IConferenceState {
     password?: string;
     passwordRequired?: Object;
     pendingSubjectChange?: string;
-    room?: Object;
+    room?: string;
```

Comment:
I suppose there was a mistake to have the room as Object, right?

Reply:
Yes